docs/getting-started.md

```diff
@@ -177,60 +190,73 @@ Claim {
   owner: '5EvSHoZF23mZS4XKQBLdqMv7a7CRSANJmxn7XDu6hwoiK4Wz' }
 ```
 
-## How to request, create and send an Attestation
+## 5. How to request, create and send an Attestation
 
-First, we need to build a request for an attestation, which has to include a claim and the address of the Claimer. (Note that this object allows much more functionality, however, we do not go into the details here):
+First, we need to build a request for an attestation, which has to include a claim and the address of the Claimer.
+(_Note_ that this object allows much more functionality, however, we do not go into the details here)
+
+### 5.1. Without privacy enhancement
+
```

Comment:
```suggestion

WIP

```

I think a little note would be nice here.

Reply:
Fixed in e1f2081